.github/workflows/ml-cicd.yml

```
@@ -297,49 +266,57 @@
 
-      - name: Download processed data
+      - name: Download DuckDB database
         uses: actions/download-artifact@v4
         with:
-          name: processed-data
-          path: src/ml/data/processed/
+          name: duckdb-dwh-${{ github.ref_name }}
+          path: src/ml/data/dwh/
 
       - name: Create Release
         uses: softprops/action-gh-release@v1
```

actionlint (1.7.7)

the runner of "softprops/action-gh-release@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)